hub/tcpclient.go

```diff
@@ -54,6 +54,56 @@ func (c *TCPClient) listen(rotator rotator.Rotator, closer chan<- *TCPClient) {
 				continue
 			}
 			rotator.SetAzimuth(az)
+
+		case "N":
+			msg = strings.TrimRight(msg[1:], "\r\n")
+
+			if len(msg) == 0 {
+				if err := c.prompt(); err != nil {
+					log.Println(err)
+					return
+				}
+				continue
+			}
+			// Elevation readout
+			el, err := strconv.Atoi(msg)
+			if err != nil {
+				log.Printf("parse error (%v): %v; msg: %s\n", c.Conn.RemoteAddr(), err, msg)
+				continue
+			}
+			rotator.SetElevation(el)
```

Comment:
I think it would be handy to check for error when calling `rotator.SetElevation`.